test/index.test.js

```diff
@@ -0,0 +1,58 @@
+const http = require ('http');
+const fs = require ('fs');
+const request = require('supertest');
+const {createServer} = http;
+const port = 5015;
+
+const {server} = require('../index');
+
+//mocking file read operation
+jest.mock('fs'), () => ({
+    readFile: jest.fn((path, callback)=> {
+        if (path === './publics/htbm.pdf') {
+            callback(null, Buffer.from('pdf'))
```

Comment:
What is the purpose of `Buffer.from('pdf')` here?

Reply:
the purpose is to load the file supposing the file path is correct